editor/components/PointMap.tsx

```diff
@@ -132,6 +138,7 @@ export const PointMap = ({
       {basemapType === "aerial" && <MapAerialSourceAndLayer />}
       <FullscreenControl position="bottom-right" />
       <NavigationControl position="top-right" showCompass={false} />
+      <MapFitBoundsControl mapRef={mapRef} bounds={geojsonBounds} />
```

Comment:
agreed about the zoom level for this crash one. Looks like adding a prop for max zoom or other parameter of the `fitBounds` call could open the door to customizing per view. These components are slick! 🚀

Reply:
added the maxZoom limit!